modules/ROOT/pages/vantagecloud-lake-demo-jupyter-docker.adoc

```diff
@@ -0,0 +1,126 @@
+= Run VantageCloud Lake Demos from a Jupyter notebook in Docker
+:experimental:
+:page-author: Vidhan Bhonsle
+:page-email: vidhan.bhonsle@teradata.com
+:page-revdate: January 10th, 2024
+:description: Run Teradata VantageCloud Lake Demos from Jupyter in a Docker
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, docker, container, vantagecloud, vantagecloud lake, lake
+
+== Overview
+In this how-to we will go through the steps for connecting to TeradataCloud Lake and run demos from a Jupyter notebook in Docker. 
+
+== Prerequisites
+* https://www.docker.com/products/docker-desktop/[Docker Desktop] installed
+* (Optional) https://git-scm.com/book/en/v2/Getting-Started-Installing-Git[Git] installed
+** Required to download git repo from https://github.com/Teradata/lake-demos.git. You can also download the repo as a .zip if you don't have Git installed locally.
+* A Teradata VantageCloud Lake account login
+** Organization URL and login details from Teradata welcome letter
+* (Optional) IDE of your choice
+
+== Create VantageCloud Lake environment
+Follow the instructions from the https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[VantageCloud Lake getting started] to create your own environment. +
+Once created, go to *SETTINGS* tab and provide your IP address range in CIDR format (for example, 192.168.2.0/24 specifies all IP addresses in the range: 192.168.2.0 to 192.168.2.255) to access the environment.
+
+image::lake_ip_addresses.png[IP whitelisting,align="center",width=50%]
+
+Your environment card should show *Public internet* access now.
+
+image::lake_public_internet_cv.png[Public internet card view,align="center",width=50%]
+
+From *OVERVIEW* tab, copy:
+
+* Public IP and
+* Open Analytics Endpoint
+
+These values are required to access VantageCloud Lake from outside.
+
+image::lake_overview_page.png[Environment Overview page,align="center",width=75%]
+
+== Clone Lake demo repository
+Clone VantageCloud Lake Demo repository in your local machine:
+
+[source, bash, id="jupyter_first_run", role="content-editable emits-gtm-events"]
+----
+git clone https://github.com/Teradata/lake-demos.git
+----
```

Comment:
Since we made the git dependency optional, we need to provide alternate instructions for users who did not install git. Alternatively, make git a hard requirement.

Reply:
For consistency, made Git a hard requirement.